### ISSUE TITLE
Regarding BOS Token and loss function

### ISSUE THREAD

**Issue by DebanjanaKar** [pengbaolin/SC-GPT#13]
Hi! Congratulations on the paper and many thanks for releasing the code for public use. I had 2 queries:
1. I find the <eos> token gets appended to the input string in line 112 of train.py but I do not find the <bos> token being used anywhere in this script. Is it getting added implicitly by any chance?

2. In the paper, in section 3, there is a mention of the loss function only attending to the utterances and not the intent during training. "During training, the prediction loss is only computed for x0, and A0 provides the attended conditions."  But I look at the labels provided for training, it includes the intent as well. Can you please help me understand how the code achieves to compute loss only for the utterance leaving out the intent?
